dash-pipeline/utils/README.md

```diff
@@ -0,0 +1,33 @@
+# DASH pipeline utils
+## Overview
+This package includes utils to configure DASH pipeline (bmv2) via p4runtime
+directly. It is a supplementary of DASH SAI, which is not doable/proper to do
+bmv2 specific configurations.
+
+## Usage
+1. internal configuration of bmv2 pipeline
+```
```

Comment:
for the markdown format issues, maybe installing the markdown-lint extension in VS code would be helpful to automatically identify and fix the issues.

Reply:
![image](https://github.com/user-attachments/assets/2c317d72-4ab0-45cf-a3d3-7e86b7f32296)

Reply:
Will do

Reply:
Fixed